site/docs/v1/tech/apis/ip-acl.adoc

```diff
@@ -0,0 +1,225 @@
+---
+layout: doc
+title: IP ACL APIs
+description: APIs for creating, retrieving, updating and deleting IP Access Control Lists
+---
+:page-liquid:
+
+== Overview
+
+An IP ACL (Access Control List) is a list of IP ranges that are either allowed or blocked. Along with one `*` or everything range that defines the default behavior. An IP ACL may be assigned to an API Key, a Tenant, or an Application. When an IP ACL is assigned at the application level this is the only one that will be used and the tenant IP ACL will not be checked.
+
+The following APIs are provided to manage IP ACLs.
+
+* <<Create an IP ACL>>
+* <<Retrieve an IP ACL>>
+* <<Search for IP ACLs>>
+* <<Update an IP ACL>>
+* <<Delete an IP ACL>>
+
+== Create an IP ACL
+
+This API is used to create a new IP ACL.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create an IP ACL with a randomly generated Id
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/ip-acl#
+--
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Create an IP ACL with the provided unique Id
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/ip-acl/`{ipAclId}`#
+--
+
+==== Request Parameters
+
+[.api]
+[field]#ipAclId# [type]#[UUID]# [optional]#Optional# [default]#defaults to secure random UUID#::
+The Id to use for the new IP ACL. If not specified a secure random UUID will be generated.
+
+include::docs/v1/tech/apis/_ip-acl-request-body.adoc[]
+
+=== Response
+
+The response for this API contains the IP ACL that was created.
+
+include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
```

Comment:
I don't think we can ever get a search engine error here, so we don't need to show a `503`. Same goes for the others as well.